HW_9/task_2.py

```diff
@@ -0,0 +1,19 @@
+class Employee:
+
+    def __init__(self, full_name: str, age: int, position: str, salary: float):
+        self.full_name = full_name
+        self.age = age
+        self.position = position
+        self.salary = salary
+
+    def description(self):
+        """The function gives the short description of the selected employee (full name, age, position and salary)."""
+        print(f'The name of the employee: {self.full_name}')
+        print(f'The age of the employee: {self.age}')
+        print(f'The position of the employee: {self.position}')
+        print(f'The salary of the employee: {self.salary}')
+
+
```

Comment:
it is not enough. 
Class should have @properties/setters
functions
staticmethods

Reply:
I still do not see properties and static methods here